### ISSUE TITLE
Script usage example

### ISSUE THREAD

**Issue by ZiadJ** [thecodewarrior/Tippy.vue#6]
Is there a demo showcasing creating and triggering the tooltip from code? Like, for instance, if one wants to display a component imported in code as a tooltip but on upon specific events like a right click event.

**Comment by thecodewarrior** [comment]
Not right now, but I could add and example of using manual triggers, however I'm not sure I understand what you mean by "display a component imported in code". Do you mean using a render function?

**Comment by ZiadJ** [comment]
Yes, a render function or it could be a component imported through the import function or referenced through ref. I like the KISS approach but also want to be sure that it is not limited to templates. Right-click functionality is also quite handy.

**Comment by thecodewarrior** [comment]
I haven't tried it in render functions since I haven't used them much and prefer using the template approach, so it isn't really designed with that as the goal, but I'll take a look. I'll also add examples of using manual triggers and context menu triggers. Depending on how much it complicates the code I may add a custom context menu trigger built-in

**Comment by ZiadJ** [comment]
Ok, thanks for the quick response. Concerning the context menu may be the official[ example from VueTippy](https://vue-tippy.netlify.app/examples/context-menu) can provide some clues.
